### DISCUSSION TITLE
How to change header depth globally

### DISCUSSION THREAD

**Discussion by wildwestrom** [rasendubi/uniorg#26] [Q&A] [answered]
How do I make it so that a top-level header `*` in org mode, be rendered as `<h2>`? How can I make it so that all heading levels in org mode are pushed down when rendered as markup?
I figured if I have `#+TITLE:`, I can use that as my `<h1>` instead.

Here's what I have in mind:
```
#+TITLE = <h1>
* <h2>
** <h3>
*** <h4>
**** <h5>
***** <h6>
****** error?
```

**Comment by rasendubi** [comment, marked as answer]
You can use something similar to this:
https://github.com/rasendubi/www.alexeyshmalko.com/blob/9140ab3251804578ed6753ead521ffd3ef54bfbb/src/lib/postprocess.ts#L41

This works on hast tree, so you should use it after uniorg-rehype.

As another option, you can implement a similar transformation on uniorg ast
by increasing the level for all headings
